2020q3/puppet.md

```diff
@@ -0,0 +1,34 @@
+## Puppet
+
+Link:	 [PuppetLab's FreeBSD slack channel](https://puppetcommunity.slack.com/messages/C6CK0UGB1/)
+Link:    [Choria](https://choria.io/)
+
+Contact: Puppet Team <puppet@FreeBSD.org>  
+
+Since out last status report a few years ago, the puppet@ team regularly
+updated the various Puppet ports to follow upstream releases of Puppet
+4, Puppet 5 and Puppet 6.  Puppet 4 was removed when it reached EOL.
```

Comment:
In general, replace all double spaces between sentences with a single space.

Reply:
Hehe, I do this without thinking about it, and find it more readable.  Since the document is converted to HTML it has no inpact in the end so I'll keep it that way for now :-)